README.md

```diff
@@ -69,10 +69,93 @@ Publishing is a two-step process:
 
 ## 🧩 Templates
 
+### Apollo Server
+```sh
+# Create a project with defaults
+npx projen new --from @ottofeller/templates ottofeller-apollo-server
+```
+```sh
+# With options
+npx projen new \
+  --from @ottofeller/templates \
+  ottofeller-apollo-server \
+  --hasDocker=false \
+  --hasGitHooks=false \
+  --lintPaths=src --lintPaths=test \
+  --hasDefaultGithubWorkflows=false
+```
+#### Options
+For a list of available options see [Common custom options](#common-custom-options).
+#### Codegen config
+See the [Codegen config](#codegen-config) subsection for the [NextJS](#nextjs) template above.
+
+### Backend test
+```sh
+npx projen new --from @ottofeller/templates ottofeller-backend-test
+```
+```sh
+# With options
+npx projen new \
+  --from @ottofeller/templates \
+  ottofeller-backend-test \
+  --hasGitHooks=false \
+  --lintPaths=src --lintPaths=test \
+  --hasDefaultGithubWorkflows=false \
+  --isGraphqlEnabled=false \
+  --isAWSDynamoDBEnabled=false
+```
+#### Options
+`isAWSDynamoDBEnabled` - sets up AWS DynamoDb dependencies and supplementary script; defaults to true.
+
+For other options see [Common custom options](#common-custom-options).
+
+### CDK
+```sh
+npx projen new --from @ottofeller/templates ottofeller-cdk
+```
+```sh
+# With options
+npx projen new \
+  --from @ottofeller/templates \
+  ottofeller-cdk \
+  --hasGitHooks=false \
+  --lintPaths=src --lintPaths=test \
+  --hasDefaultGithubWorkflows=false \
+  --hasRustTestWorkflow=true \
+  --isGraphqlCodegenEnabled=true \
+  --initialReleaseVersion='1.0.0'
+```
+#### Options
+`initialReleaseVersion` - the base version of the very first release; defaults to `0.0.1`.
+
+`isGraphqlCodegenEnabled` - sets up GraphQL dependencies and supplementary script; defaults to  `false`.
+
+For other options see [Common custom options](#common-custom-options).
+
 ### NextJS
 ```sh
 npx projen new --from @ottofeller/templates ottofeller-nextjs
 ```
+```sh
+# With options
+npx projen new \
+  --from @ottofeller/templates \
+  ottofeller-nextjs \
```

Comment:
@quesabe nit stuff, can we move the project name to the very top, or very bottom of the list of params?